soh/soh/Enhancements/randomizer/3drando/logic.cpp

```diff
@@ -292,8 +293,8 @@ namespace Logic {
   bool CanUseMagicArrow = false;
 
   //Bridge and LACS Requirements
-  uint8_t MedallionCount          = 0;
   uint8_t StoneCount              = 0;
+  uint8_t MedallionCount          = 0;
```

Comment:
Is there any particular reason why Stones and Medallions were re-ordered across the code base like this? If it is not critical, I would prefer if the ordering was left untouched as the change is increasing the noise in this PR and is making our version of the 3ds code harder to compare with theirs.

Reply:
I changed it to line up with the bridge requirement order. It doesn't make sense to have bridge requirements go stones first, then medallions, but have lacs be opposite. I didn't figure this minor change would need a separate PR, or be a big deal, but I can change it all back if that's preferred.

Reply:
Ok, I missed that it differed between the two. I thought you were reordering both. Then this is fine to me, it is not a lot of changes.